src/src/Helpers/ESPEasy_Storage.cpp

```diff
@@ -1144,6 +1144,44 @@ String LoadTaskSettings(taskIndex_t TaskIndex)
   return result;
 }
 
+#if FEATURE_ALTERNATIVE_CDN_URL
+static String _CDN_url_cache;
+static bool   _CDN_url_loaded = false;
+
+String get_CDN_url_override() {
+  if (!_CDN_url_loaded) {
+    char override[256]{};
+    const __FlashStringHelper* cdnOverrideFile = F("cdnoverride.dat");
```

Comment:
We do have a set of 'system files' with their filetype and functions to/from the file name.
Why not extend that with a new type for the CDN_location?
And I don't think "override" is a good name.
Maybe "Custom_CDN.txt" is a better name?

Reply:
I tried to avoid the .txt extension to discourage users from downloading/editing the file manually (though that's quite hard already), and not have them insert comments etc. The UI is the place to edit this file.
And I'm looking at the Provisioning feature, how to handle it there (ignoring that for now, to be implemented though)